src/openrct2/ride/Vehicle.cpp

```diff
@@ -1392,7 +1394,7 @@ static bool vehicle_close_restraints(rct_vehicle* vehicle)
                 continue;
             }
         }
-        vehicle_invalidate(vehicle);
+        vehicle->Invalidate();//updated function call to match becoming a member function of rct_vehicle struct
```

Comment:
Please remove this comment and the others like it.

Reply:
oops my bad. i actually meant to take those out but forgot